modules/ROOT/pages/clauses/merge.adoc

```diff
@@ -683,3 +683,55 @@ RETURN person.name, person.bornIn, person.chauffeurName
 | person.name | person.bornIn | person.chauffeurName
 | "Keanu Reeves" | "Beirut" | "Eric Brown"
 |===
+
+[role=label--new-5.26]
+[[dynamic-merge]]
+== MERGE nodes and relationships using dynamic node labels and relationship types
+
+Node labels and relationship types can be referenced dynamically in parameters and variables when merging nodes and relationships.
+
+.Syntax for merging nodes and relationships dynamically
+[source, syntax]
+----
+MERGE (n:$(<expr>)),
+MERGE ()-[r:$(<expr>)]->()
+----
+
+The expression must evaluate to a `STRING NOT NULL | LIST<STRING NOT NULL> NOT NULL` value.
```

Comment:
Would be good to mention maybe that relationships can still only have one type, even if a list is supplied